test/runtests.jl

```diff
@@ -4,5 +4,11 @@ using Test
 
 pomdp = TigerPOMDP()
 write(stdout, pomdp)
-m1 = read_pomdp("1d.pomdp")
-m2 = read_pomdp("parr95.95.pomdp")
+f1 = tempname()
+f2 = tempname()
+download("http://pomdp.org/examples/1d.POMDP", f1)
+m1 = read_pomdp(f1)
+println(m1.discount == 0.75 && m1.O[1,1,1] == 1.0 && m1.T[1,1,1] == 1.0 && m1.R[1,1] == 1.0)
```

Comment:
Instead of `println` here, please replace with `@test` as done in other packages.